packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -295,6 +296,7 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
           isSelected={selectedIndex === __originalIndex}
           onClick={() => dispatch(['select', __originalIndex])}
           index={__originalIndex}
+          data-iui-filtered-index={filteredIndex}
```

Comment:
Just thinking, can this attribute be used to simplify regular keyboard navigation?

Reply:
Ok, maybe it doesn't help